### ISSUE TITLE
Can't browse history when other users are messaging

### ISSUE THREAD

**Issue by monofuel** [cryptag/leapchat#32]
If you are scrolled up and someone makes a new chat message, it automatically scrolls you down. this can make it difficult to view chat history if other people are talking.

**Comment by elimisteve** [comment]
Good point. I think Slack just tells you that there's new messages but doesn't scroll you down, which is a solution you just mentioned in person; sounds like a good solution.
